db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/sqlgrammar.jj

```diff
@@ -7405,22 +7405,47 @@ characterValueFunction() throws StandardException :
         return getReplaceNode( value, str1, str2 );
 	}
 |
-	( upperTok = <UPPER> | lowerTok = <LOWER> ) <LEFT_PAREN> value = additiveExpression(null,0) <RIGHT_PAREN>
+	( upperTok = <UPPER> | lowerTok = <LOWER> ) <LEFT_PAREN> str1 = additiveExpression(null,0) [ <COMMA> str2 = additiveExpression(null,0) ] <RIGHT_PAREN>
```

Comment:
Please use more descriptive names, I think str1 is value and str2 locale, is that right?

Reply:
@dgomezferro Pushed new commit